test/unit/tools/test_optools.py

```diff
@@ -388,36 +380,35 @@ def setUp(self):
             [ 0.71538573+0.j,  0.2680266 +0.36300238j, 0.2680266 -0.36300238j,  0.28461427+0.j]])
 
     def test_jtrace_distance(self):
-        self.assertAlmostEqual(ot.jtracedist(self.A, self.A, mx_basis="std"), 0.0)
-        self.assertAlmostEqual(ot.jtracedist(self.A, self.B, mx_basis="std"), 0.26430148)  # OLD: 0.2601 ?
+        val = ot.jtracedist(self.A_TP, self.A_TP, mx_basis="pp")
+        self.assertAlmostEqual(val, 0.0)
+        val = ot.jtracedist(self.A_TP, self.B_unitary, mx_basis="pp")
+        self.assertGreaterEqual(val, 0.5)
 
     @needs_cvxpy
     def test_diamond_distance(self):
         if SKIP_DIAMONDIST_ON_WIN and sys.platform.startswith('win'): return
-        self.assertAlmostEqual(ot.diamonddist(self.A, self.A, mx_basis="std"), 0.0)
-        self.assertAlmostEqual(ot.diamonddist(self.A, self.B, mx_basis="std"), 0.614258836298)
+        val = ot.diamonddist(self.A_TP, self.A_TP, mx_basis="pp")
+        self.assertAlmostEqual(val, 0.0)
+        val = ot.diamonddist(self.A_TP, self.B_unitary, mx_basis="pp")
+        self.assertGreaterEqual(val, 0.7)
 
     def test_entanglement_fidelity(self):
-        fidelity = ot.entanglement_fidelity(self.A, self.B)
```

Comment:
Why was this test removed?

Reply:
``A`` and ``B`` are psd matrices, not operation matrices. If you try to get their Choi matrix representations then you get a non-Hermitian matrix as a result, which leads to an error (correctly) getting thrown by the new implementation of fidelity.